export_as_latex.ml

```diff
@@ -17,7 +17,8 @@ let proof_to_latex implicit_exchange notations proof =
         ^ "\\newcommand*{\\zero}{0}\n"
         ^ "\\newcommand*{\\limp}{\\multimap}\n\n"
         ^ "\\newcommand*{\\hypv}[1]{\\hypo{\\vdash #1}}\n"
-        ^ "\\newcommand*{\\exv}[1]{\\infer{1}[\\ensuremath{\\mathit{ex}}]{\\vdash #1}}\n"
+        ^ "\\newcommand*{\\exv}[2]{\\infer{1}[\\ensuremath{\\mathit{ex}}]{\\vdash #2}}\n"
+        ^ "\\newcommand*{% \\exv}[2]{\\infer{1}[\\ensuremath{\\mathit{ex}_{#1}}]{\\vdash #2}}\n"
```

Comment:
```suggestion
        ^ "% \\newcommand*{\\exv}[2]{\\infer{1}[\\ensuremath{\\mathit{ex}_{#1}}]{\\vdash #2}}\n"
```

Reply:
sorry...

Reply:
I let you commit it I don't have rights to do it on your branch.